package.json

```diff
@@ -67,5 +67,8 @@
     "notation",
     "guitar",
     "tablature"
-  ]
+  ],
+  "dependencies": {
+    "es-dev-server": "^1.54.1"
```

Comment:
This should be in `devDependencies`, but also it's not reference from any scripts above. Are you still using it?

Reply:
I've just been using it directly in my terminal to run, so I didn't put it in a script. Is it better to put it in a script?

Reply:
Yeah, especially so others have an easier time knowing how to use it.